libkbfs/repeated_wait_group.go

```diff
@@ -20,6 +20,12 @@ type RepeatedWaitGroup struct {
 	lock     sync.Mutex
 	num      int
 	isIdleCh chan struct{} // leave as nil when initializing
+	// TODO: we could remove this paused bool by converting the
```

Comment:
Thanks for the suggestion, but this seems just as complicated to me but not as intuitive, plus the var names are longer.  I vote to keep it as is.